### ISSUE TITLE
ManualValidation comments as output variable

### ISSUE THREAD

**Issue by lion92pl** [microsoft/azure-pipelines-tasks#14449]
## Note  
Issues in this repo are for tracking bugs, feature requests and questions for the tasks in this repo 

For a list:  
https://github.com/Microsoft/azure-pipelines-tasks/tree/master/Tasks

If you have an issue or request for the Azure Pipelines service, use developer community instead:  

https://developercommunity.visualstudio.com/spaces/21/index.html )

## Required Information

Entering this information will route you directly to the right team and expedite traction.

**Question, Bug, or Feature?**  
*Type*: Feature

**Enter Task Name**: ManualValidation

list here (V# not needed):  
https://github.com/microsoft/azure-pipelines-tasks/tree/master/Tasks/ManualValidationV0

## Issue Description

Manual validation task provides field for comments in UI.
![image](https://user-images.githubusercontent.com/17692934/108844742-cc8d4f80-75dc-11eb-9c85-ed90b3bba20e.png)
I would like to use this value inside next tasks/jobs/stages in pipeline.
It is not returned as output variable or it is not documented.

**Comment by chshrikh** [comment]
@ds-ms please assign it appropriately

**Comment by andmig-ilty** [comment]
That would be really great! In general it'd be great to have some sort of "data entry" facilities for already running pipelines.

Since ManualValidation must be in a Agentless Job, that output variable would most likely be read from other jobs with dependsOn relationship.

**Comment by tomasz-soltysik** [comment]
Any update?

**Comment by alexstdev** [comment]
Yes please, it would be great to get this information, to be able to archive it in the following step.

Also, once you rerun a rejected manual validation job, the comment disappears and you can no longer read the reason as to why it got failed in the first place.

**Comment by trevonmckay** [comment]
We recently moved from Classic Releases to Multi-Stage YAML pipelines in our PR workflows. Ideally, we'd like to take the comments from the Manual Validation task and write them back to the PR that triggered the pipeline if the Manual Validation was rejected.

**Comment by chielboogaard** [comment]
any update on this?

i would also like this feature. Also i would like to get the user that clicked the resume button.

**Comment by wyrdfish** [comment]
I too have been looking for a way to use this value in a report. 
It could also help if it was exposed in a rest api against the run.

**Comment by piyaldas** [comment]
Any update on this?
We are also looking for this feature to pass the details from Manual Validation to next stage to track down every information.

**Comment by wyrdfish** [comment]
I added a step to the build that uses the web api to read the details.
https://gist.github.com/wyrdfish/a68c678dc344c554ebbb69e61754993b
based on this https://blogs.blackmarble.co.uk/rfennell/getting-the-approver-for-release-to-an-environment-within-an-azure-devops-multi-stage-yaml-pipeline/

**Comment by limberger** [comment]
Any update? This feature would be very useful. Or any way to enter additional data during the pipeline execution.

**Comment by jmakuc** [comment]
@wyrdfish workaround is great, but the task should output at least the user and the comment.

**Comment by Kookitanou92** [comment]
Any Updates on this ?

**Comment by Warns** [comment]
This feature would be great indeed
